app/slackimport.go

```diff
@@ -786,11 +786,10 @@ func (a *App) OldImportUser(team *model.Team, user *model.User) *model.User {
 }
 
 func (a *App) OldImportChannel(channel *model.Channel) *model.Channel {
-	result := <-a.Srv.Store.Channel().Save(channel, *a.Config().TeamSettings.MaxChannelsPerTeam)
```

Comment:
nit: Here you can just use `return a.Srv.Store.Channel().Save(channel, *a.Config().TeamSettings.MaxChannelsPerTeam)`